bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -105,34 +109,86 @@ public Builder setShortRpcTimeoutMs(int shortRpcTimeoutMs) {
      *
      * @param longRpcTimeoutMs timeout value in milliseconds.
      * @return a {@link Builder} object, for chaining
+     * @deprecated Please use {@link #setMutateRpcTimeoutMs(int)} or {@link
+     *     #setReadRowsRpcTimeoutMs(int)}.
      */
+    @Deprecated
     public Builder setLongRpcTimeoutMs(int longRpcTimeoutMs) {
       Preconditions.checkArgument(longRpcTimeoutMs > 0, "Long Timeout ms has to be greater than 0");
       this.longRpcTimeoutMs = longRpcTimeoutMs;
       return this;
     }
 
+    /**
+     * The amount of milliseconds to wait before issuing a client side timeout for mutation RPCs.
+     *
+     * @param mutateRpcTimeoutMs timeout value in milliseconds.
+     * @return a {@link Builder} object, for chaining
+     */
+    public Builder setMutateRpcTimeoutMs(int mutateRpcTimeoutMs) {
+      Preconditions.checkArgument(
+          mutateRpcTimeoutMs > 0, "Long Timeout ms has to be greater than 0");
```

Comment:
should mutateRpcTimeoutMs be an int or a long?

Reply:
I think "Long Timeout" referred to "long rpc" vs. "short rpc" timeouts.  Timeouts should be `int`s.

Reply:
@sduskis Thanks for the explanation

Reply:
Thanks! In that case, can we add that clarification to the error message? That wasn't obvious to me when first reading.